cmake/Config.cmake.in

```diff
@@ -1,7 +1,7 @@
 include(CMakeFindDependencyMacro)
 
 # Tell library users about the Boost dependency
-find_dependency(Boost 1.53)
+find_dependency(Boost 1.53 HINTS ${CMAKE_CURRENT_LIST_DIR}/..)
 
 # Add the targets file
 include("${CMAKE_CURRENT_LIST_DIR}/YOMM2Targets.cmake")
```

Comment:
EOL please.

Reply:
Thank you for the feedback! 
On my machine it works without root privilege (in fact I never use root when developing). 
Are you specifying an install directory for yomm? It looks like it is trying to install in `/use/local`. The default I have set is to install in a subdirectory of the source tree.

I'll fix the simple EOL stuff first and then look into what is going on here.
Could you post the output of the cmake command you run as root?

Reply:
done